cmd/statusd-prune/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/status-im/status-go/mailserver"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+var (
+	dbPath         = flag.String("db", "", "Path to wnode database folder")
+	lowerTimestamp = flag.Int("lower", 0, "Removes messages sent starting from this timestamp")
+	upperTimestamp = flag.Int("upper", 0, "Removes messages sent up to this timestamp")
+)
+
+func missingFlag(f string) {
+	log.Printf("flag -%s is required", f)
+	flag.Usage()
+	os.Exit(1)
+}
+
+func init() {
+	flag.Parse()
+
+	if *dbPath == "" {
+		missingFlag("db")
```

Comment:
`-db`

Reply:
I see it now. So the one below is invalid then: `missingFlag("-upper")`?

Reply:
yes you are right :)

Reply:
I updated `upper`